.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+# Introduction
+
+**KIN welcomes all contributions**. We're building a product that will hopefully make the world a fairer place,
+as such we could not build it without the efforts of many people,
+so please don't be shy and share your help with us. Even the tiniest contribution can make a difference!
+
+These guidelines briefly explain how to contribute in an effective manner, making sure to keep high
+quality standards and making it easier for your contributions to make through.
+
+There are many ways to contribute, from writing tutorials or [blog posts](https://medium.com/kinblog), improving the [documentation](https://partners.kinecosystem.com), asking and answering questions on [stackoverflow](https://stackoverflow.com/questions/tagged/kin), submitting bug reports and feature requests or writing code.
+
+Kin has many public [Github repositories](https://github.com/kinecosystem) for an arrary of products. Before openning an issue, please make sure you are in the right reposity. 
+
+If you have any product questions please join [our community on reddit](https://www.reddit.com/r/KinFoundation/).
+
+Note that we have a code of conduct, which we ask you follow in all your interactions with the project.
+
+## Getting Started
+
+1. Create your own fork of the code
+1. Do the changes in your fork
+1. Be sure you have followed the code style for the project
+1. Make sure you add unit tests to any new code you wrote
+1. Send a pull request
+
+## How to report a bug
+
+If you find a security vulnerability, do NOT open an issue. Email security@kinecosystem.com instead.
+
+When filing an issue, make sure to answer these questions:
```

Comment:
why all have number `1`?

Reply:
Markdown will convert it to relevant numbers